docs/eigenda/rollup-guides/tutorial.md

```diff
@@ -53,7 +53,7 @@ cd eigenda
 
 grpcurl -import-path ./api/proto -proto ./api/proto/disperser/disperser.proto -d
 '{"data": "0000", "security_params": [{"quorum_id": 0, "adversary_threshold":
-25, "quorum_threshold": 50}]}' disperser-goerli.eigenda.xyz:443
+25, "quorum_threshold": 50}]}' disperser-holesky.eigenda.xyz:443
```

Comment:
When is this doc change expected to hit the prod? This disperser URL shouldn't be reachable until testnet is launched to rollups (few days after operators onboard)

Reply:
Rather than do a really complicated docs change where we take down old docs, then add operator docs, then add rollup docs let's just simplify by updating everything right now. It won't confuse things too much.